frontend/package.json

```diff
@@ -13,13 +13,17 @@
   "dependencies": {
     "@emotion/react": "^11.10.5",
     "@emotion/styled": "^11.10.5",
+    "@hookform/resolvers": "^2.9.11",
     "@mui/icons-material": "^5.11.0",
     "@mui/material": "^5.11.6",
     "@vitejs/plugin-react-refresh": "^1.3.6",
+    "axios": "^1.3.3",
```

Comment:
As discussed We should just use fetch

Reply:
Fixed. Now `fetch` is used instead of `axios`.